ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -0,0 +1,415 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  Flex,
+  Box,
+  HStack,
+  Button,
+  ButtonGroup,
+  VStack,
+  Center,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  Container,
+  AccordionIcon,
+} from "@chakra-ui/react";
+import { SocialIcon } from "react-social-icons";
+import "./main-menu.css";
+
+const MainMenu = ({ login, view }) => {
+  const WebView = () => {
+    return (
+      <Container maxW="container.lg">
+        <HStack
+          marginLeft="40px"
+          display={{ base: "none", md: "flex" }}
+          color="white"
+        >
+          <Flex marginTop="30px">
+            <VStack alignItems="flex-start">
+              <h1>Visit</h1>
+              <Link href="/">
```

Comment:
hyperlinks should be configured to existing pages as much as possible.

Reply:
connected to most existing pages (login and events page) for now